### ISSUE TITLE
Data Science with Python : LSTM Artificial Neural Networks

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#4159]
Welcome to 'DSWP' Team, good to see you here

With this issue, readers will get introduced with core information to LSTM Artificial Neural Networks completely in a effective way.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Artificial Intelligence

Mentors Assigned : Nithin / Dhruv Bajaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by ron352** [comment]
Sarna
serial no :614
type : audio 
@prathimacode-hub  mam can i work on this?

**Comment by prathimacode-hub** [comment]
This is created on contributor's request. If not claimed by the time, it will be assigned to next on queue. @ron352

**Comment by ron352** [comment]
Ok mam

**Comment by aswinikalyan30** [comment]
Good evening ma'am,  I would like to work on "LSTM Artificial Neural Network" for documentation 
Name: Aswini Kalyan 
Serial Number: 406
Batch: 11
Type: Documentation 
Approach: 
-Introduction
-Variations
-Model training on stock price as example
-Applications
Github: https://github.com/aswinikalyan30

**Comment by prathimacode-hub** [comment]
Issue assigned to @aswinikalyan30 for documentation

**Comment by ron352** [comment]
@prathimacode-hub mam is its audio section is still on?
Can I work on that?

**Comment by prathimacode-hub** [comment]
Yes, it's available. Issue assigned to @ron352 for audio contribution

**Comment by ashish-reddy-20-08** [comment]
@prathimacode-hub , Mam
I am M. Ashish Reddy. I would like to do video on this issue.
Batch number: 5
serial number:174.
Approach: A detailed explanation with examples and real life uses

**Comment by prathimacode-hub** [comment]
Issue assigned to @ashish-reddy-20-08 for video contribution

**Comment by deepthi1107** [comment]
Hello, I am Deepthi M with serial number:172, Batch-5. I would like to do audio on this issue.
My approach: a detailed explanation

**Comment by prathimacode-hub** [comment]
Issue assigned to @deepthi1107 for audio contribution
